src/billing/components/BillingPageContents.tsx

```diff
@@ -4,13 +4,18 @@ import React, {FC} from 'react'
 // Components
 import BillingFree from 'src/billing/components/Free/Free'
 import BillingPayAsYouGo from 'src/billing/components/PayAsYouGo/PayAsYouGo'
+import MarketplaceBilling from 'src/billing/components/marketplace/MarketplaceBilling'
 
 // Utils
 import {useBilling} from 'src/billing/components/BillingPage'
 
 const BillingPageContents: FC = () => {
   const [{account}] = useBilling()
 
+  if (account.type === 'pay_as_you_go' && account.marketplace !== 'Zuora') {
```

Comment:
I think this is OK, but in Quartz, we only look at whether the account has a marketplace subscription or not. I think the equivalent here would be `account.marketplace !== 'Zuora'` without the `pay_as_you_go` check.